--- Makefile	2016-07-31 10:21:16.000000000 -0300
+++ Makefile.comp	2016-07-31 11:07:56.000000000 -0300
@@ -82,11 +82,6 @@
 	@export $$(cat .env | xargs) && echo "DRUPAL_BOILERPLATE_SSH_PORT="$$DRUPAL_BOILERPLATE_SSH_PORT
 	@export $$(cat .env | xargs) && echo "DRUPAL_BOILERPLATE_DOWNLOAD_VERSION="$$DRUPAL_BOILERPLATE_DOWNLOAD_VERSION
 
-
-download:
-	@read -p "Do you wish download Drupal? You will lost everything inside a public directory [y/n] : " yn && test $$yn == 'y' && exit 0
-	@export $$(cat .env | xargs) && cd infrastructure/environments/${ENVIRONMENT}/docker && rm -Rf ${DRUPAL_BOILERPLATE_FOLDER}/* && curl https://ftp.drupal.org/files/projects/drupal-$${DRUPAL_BOILERPLATE_DOWNLOAD_VERSION}.tar.gz | tar zx && cp -Rf drupal-$${DRUPAL_BOILERPLATE_DOWNLOAD_VERSION}/* ${DRUPAL_BOILERPLATE_FOLDER}/ && rm -Rf drupal-$${DRUPAL_BOILERPLATE_DOWNLOAD_VERSION}
-
 test_server_up:
 	@export $$(cat .env | xargs) && cd infrastructure/environments/${ENVIRONMENT}/docker && docker-compose -f docker-compose.test.yml up -d
 	@sleep 3
@@ -109,6 +104,3 @@
 
 test_help:
 	@join -v 1 <(cat Makefile | grep ^[a-z][^\w]*: | cut -f1 -d":" | sort) <(make help | cut -f2 -d" " | sort)
-
-travis: set_variable all
-	@export $$(cat .env | xargs) && cd infrastructure/environments/${ENVIRONMENT}/docker && docker-compose --project-name $${DRUPAL_BOILERPLATE_PROJECT_NAME}_$${DRUPAL_BOILERPLATE_DRUPAL_VERSION} -f docker-compose.d$${DRUPAL_BOILERPLATE_DRUPAL_VERSION}.yml stop && docker-compose --project-name $${DRUPAL_BOILERPLATE_PROJECT_NAME}_$${DRUPAL_BOILERPLATE_DRUPAL_VERSION} -f docker-compose.d$${DRUPAL_BOILERPLATE_DRUPAL_VERSION}.yml rm -f
